source/docs/api-reference/wpilib-integration/motorcontroller-integration.rst

```diff
@@ -0,0 +1,69 @@
+MotorController Integration
+===========================
+
+Phoenix Pro motor controller classes such as ``TalonFX`` (`Java <https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/hardware/TalonFX.html>`__, `C++ <https://api.ctr-electronics.com/phoenixpro/release/cpp/classctre_1_1phoenixpro_1_1hardware_1_1_talon_f_x.html>`__) implement the ``MotorController`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/motorcontrol/MotorController.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc_1_1_motor_controller.html>`__) interface. This allows Phoenix Pro motor controllers to be used in WPILib drivetrain classes such as ``DifferentialDrive``.
+
+.. tab-set::
+
+   .. tab-item:: Java
+      :sync: Java
+
+      .. code-block:: java
+
+         // instantiate motor controllers
+         TalonFX m_motorLeft = new TalonFX(0);
+         TalonFX m_motorRight = new TalonFX(1);
+
+         // create differentialdrive object for robot control
+         DifferentialDrive m_diffDrive = new DifferentialDrive(m_motorLeft, m_motorRight);
+
+         // instantiate joystick
+         XboxController m_driverJoy = new XboxController(0);
+
+         public void teleopPeriodic() {
+            var forward = -m_driverJoy.getLeftY();
+            var rot = -m_driverJoy.getRightX();
+
+            m_diffDrive.arcadeDrive(forward, rot);
+         }
+
+   .. tab-item:: C++ (Source)
+      :sync: C++
+
+      .. code-block:: cpp
+
+         void Robot::TeleopPeriodic() {
+            auto forward = -m_driverJoy.GetLeftY();
+            auto rot = -m_driverJoy.GetRightX();
+
+            m_diffDrive.ArcadeDrive(forward, rot);
+         }
+
+   .. tab-item:: C++ (Header)
+      :sync: C++ Header
+
+      .. code-block:: cpp
+
+         // instantiate motor controllers
+         hardware::TalonFX m_motorLeft{0};
+         hardware::TalonFX m_motorRight{1};
+
+         // create differentialdrive object for robot control
+         frc::DifferentialDrive m_diffDrive{m_motorLeft, m_motorRight};
+
+         // instantiate joystick
+         frc::XboxController m_driverJoy{0};
+
+Motor Safety
+------------
+
+CTR-Electronics supported actuators implement WPILib `Motor Safety <https://docs.wpilib.org/en/stable/docs/software/hardware-apis/motors/wpi-drive-classes.html#motor-safety>`__. Motor Safety (in addition to the normal timeout features of CTR-Electronics actuators) will automatically disable the device every 5 received packets (~100ms) if ``feed()`` has not been called. ``feed()`` is automatically called when using any of the motor controller ``set`` routines (``setControl``, ``set``, ``DifferentialDrive.arcadeDrive``, etc). Please see the relevant `Motor Safety <https://docs.wpilib.org/en/stable/docs/software/hardware-apis/motors/wpi-drive-classes.html#motor-safety>`__ documentation for more information.
```

Comment:
I wouldn't copy any of the WPILib Motor Safety docs here in case it changes. First sentence is fine, replace the rest with something like:

"In addition to the normal :doc:`enable signal <enabling actuators link>` of CTR-Electronics actuators, Motor Safety will automatically disable the device according to the WPILib Motor Safety implementation.